ff/serving/check_coverage.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Unrelated to this file, but add .idea to .gitignore

Reply:
Added